apps/web/src/pages/ApplicantDashboardPage/components/ReviewApplicationPreviewList.tsx

```diff
@@ -0,0 +1,578 @@
+} from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+import talentRequestMessages from "~/messages/talentRequestMessages";
+import processMessages from "~/messages/processMessages";
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  ApplicationStatus,
+  getApplicationStatusChip,
+  getSalaryRange,
+} from "~/utils/poolCandidate";
+import { sortPoolSkillsBySkillCategory } from "~/utils/skillUtils";
+import useRoutes from "~/hooks/useRoutes";
+
+import { ApplicationDate } from "./MetadataDate";
+import StatusSummary from "./StatusSummary";
+
+export const ReviewApplicationPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewApplicationPreviewList on PoolCandidate {
+    id
+    expiryDate
+    finalDecisionAt
+    submittedAt
+    removedAt
+    status {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    finalDecision {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    assessmentStatus {
+      assessmentStepStatuses {
+        step
+      }
+      overallAssessmentStatus
+      currentStep
+    }
+    pool {
+      id
+      name {
+        en
+        fr
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+      workStream {
+        name {
+          en
+          fr
+        }
+      }
+      language {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      department {
+        name {
+          en
+          fr
+        }
+      }
+      closingDate
+      opportunityLength {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      isRemote
+      location {
+        en
+        fr
+      }
+      securityClearance {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      areaOfSelection {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      processNumber
+      poolSkills {
+        id
+        type {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        skill {
+          id
+          key
+          name {
+            en
+            fr
+          }
+          category {
+            value
+            label {
+              en
+              fr
+            }
+          }
+        }
+      }
+      screeningQuestions {
+        id
+      }
+    }
+  }
+`);
+
+interface ReviewApplicationDialogProps {
+  applicationQuery: ReviewApplicationPreviewListFragment;
+}
+
+const ReviewApplicationDialog = ({
+  applicationQuery,
+}: ReviewApplicationDialogProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useRoutes();
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const application = applicationQuery;
```

Comment:
If you are just aliasing this you can do it in the param destructing:

```
({ applicationQuerty: application })
```

Reply:
Changed in [b107fc6](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/b107fc6b2eec3d0bb76c0655b4b138371675889a)